File: CS/WinWebSolution.Web/ApplicationCode/WebApplication.cs
using System;
using DevExpress.ExpressApp.Web;
using DevExpress.ExpressApp;

namespace DcDemo.Web {
    public class WinWebSolutionAspNetApplication : WebApplication {
		private DevExpress.ExpressApp.SystemModule.SystemModule systemModule1;
		private DevExpress.ExpressApp.Web.SystemModule.SystemAspNetModule webSystemModule1;
		private DevExpress.ExpressApp.Validation.ValidationModule validationModule1;
		private WinWebSolution.Module.Web.WinWebSolutionAspNetModule module1;
        private WinWebSolution.Module.WinWebSolutionModule module2;
    
		public WinWebSolutionAspNetApplication() {
			InitializeComponent();
		}
        private void InitializeComponent() {
			System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(WinWebSolutionAspNetApplication));
			this.systemModule1 = new DevExpress.ExpressApp.SystemModule.SystemModule();
			this.webSystemModule1 = new DevExpress.ExpressApp.Web.SystemModule.SystemAspNetModule();
			this.validationModule1 = new DevExpress.ExpressApp.Validation.ValidationModule();
            this.module2 = new WinWebSolution.Module.WinWebSolutionModule();
            this.module1 = new WinWebSolution.Module.Web.WinWebSolutionAspNetModule();
			((System.ComponentModel.ISupportInitialize)(this)).BeginInit();
            this.ApplicationName = "WinWebSolution";
			this.Modules.Add(this.systemModule1);
			this.Modules.Add(this.webSystemModule1);
			this.Modules.Add(this.validationModule1);
			this.Modules.Add(this.module2);
			this.Modules.Add(this.module1);
			this.DatabaseVersionMismatch += new System.EventHandler<DevExpress.ExpressApp.DatabaseVersionMismatchEventArgs>(this.DcDemoWebApplication_DatabaseVersionMismatch);
			((System.ComponentModel.ISupportInitialize)(this)).EndInit();

        }
        private void DcDemoWebApplication_DatabaseVersionMismatch(object sender, DatabaseVersionMismatchEventArgs e) {
            e.Updater.Update();
            e.Handled = true;
        }
	}
}

File: VB/WinWebSolution.Web/ApplicationCode/WebApplication.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp.Web
Imports DevExpress.ExpressApp

Namespace DcDemo.Web
	Public Class WinWebSolutionAspNetApplication
		Inherits WebApplication
		Private systemModule1 As DevExpress.ExpressApp.SystemModule.SystemModule
		Private webSystemModule1 As DevExpress.ExpressApp.Web.SystemModule.SystemAspNetModule
		Private validationModule1 As DevExpress.ExpressApp.Validation.ValidationModule
		Private module1 As WinWebSolution.Module.Web.WinWebSolutionAspNetModule
		Private module2 As WinWebSolution.Module.WinWebSolutionModule

		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub InitializeComponent()
			Dim resources As New System.ComponentModel.ComponentResourceManager(GetType(WinWebSolutionAspNetApplication))
			Me.systemModule1 = New DevExpress.ExpressApp.SystemModule.SystemModule()
			Me.webSystemModule1 = New DevExpress.ExpressApp.Web.SystemModule.SystemAspNetModule()
			Me.validationModule1 = New DevExpress.ExpressApp.Validation.ValidationModule()
			Me.module2 = New WinWebSolution.Module.WinWebSolutionModule()
			Me.module1 = New WinWebSolution.Module.Web.WinWebSolutionAspNetModule()
			CType(Me, System.ComponentModel.ISupportInitialize).BeginInit()
			Me.ApplicationName = "WinWebSolution"
			Me.Modules.Add(Me.systemModule1)
			Me.Modules.Add(Me.webSystemModule1)
			Me.Modules.Add(Me.validationModule1)
			Me.Modules.Add(Me.module2)
			Me.Modules.Add(Me.module1)
'			Me.DatabaseVersionMismatch += New System.EventHandler(Of DevExpress.ExpressApp.DatabaseVersionMismatchEventArgs)(Me.DcDemoWebApplication_DatabaseVersionMismatch);
			CType(Me, System.ComponentModel.ISupportInitialize).EndInit()

		End Sub
		Private Sub DcDemoWebApplication_DatabaseVersionMismatch(ByVal sender As Object, ByVal e As DatabaseVersionMismatchEventArgs) Handles MyBase.DatabaseVersionMismatch
			e.Updater.Update()
			e.Handled = True
		End Sub
	End Class
End Namespace
